pkgs/top-level/python-packages.nix

```diff
@@ -25797,4 +25797,21 @@ in modules // {
     };
   };
 
+  topydo = buildPythonPackage rec {
+    name = "topydo-${version}";
+    version = "0.9";
+    disabled = (!isPy3k);
+
+    src = pkgs.fetchFromGitHub {
+      owner = "bram85";
+      repo = "topydo";
+      rev = version;
+      sha256 = "0vmfr2cxn3r5zc0c4q3a94xy1r0cv177b9zrm9hkkjcmhgq42s3h";
+    };
+
+    propagatedBuildInputs = with self; [ arrow icalendar ];
+
+    doCheck = false;
```

Comment:
I got a question about this: I tried to get the tests working, but it required to add two dependencies which weren't in nixpkgs, and it still isn't working.

So should I add the dependencies to nixpkgs and keep them commented in the derivation or just ignore them as they aren't used anyway ?